tests/test_Summary_GSatProd.cpp

```diff
@@ -0,0 +1,1354 @@
+/*
+  Copyright 2016 Statoil ASA.
```

Comment:
should probably be updated

Reply:
> should probably be updated

:flushed:

Good catch!  I'll fix that.